src/components/scenes/SendScene2.tsx

```diff
@@ -1025,8 +1025,7 @@ const SendComponent = (props: Props) => {
       {({ insetStyle }) => (
         <>
           <StyledKeyboardAwareScrollView
-            notificationHeight={insetStyle.paddingBottom}
-            contentContainerStyle={{ ...insetStyle, paddingTop: 0, paddingBottom: theme.rem(5) + insetStyle.paddingBottom }}
+            contentContainerStyle={{ ...insetStyle, paddingTop: 0 }}
```

Comment:
This ignores the padding needed to ensure the slider doesn't overlap the content. That's what the 5 rem was for in the first place. It wasn't for the notification card. Try adding 5 target addresses and see what happens

Reply:
The screenshot I posted as a code comment was meant to illustrate this - the slider does not cover any content already and the extra 5 rem isn't needed

Reply:
I just tested the code and here's a screenshot with the lowest tile occluded verifying my comment

<img width="515" alt="Screenshot 2024-03-15 at 17 55 29" src="https://github.com/EdgeApp/edge-react-gui/assets/6079354/be7130bf-5eb7-4470-96f0-9a22fa8d0594">

Reply:
![image](https://github.com/EdgeApp/edge-react-gui/assets/90650827/d307647c-1a8c-4497-86f5-629761430573)
![image](https://github.com/EdgeApp/edge-react-gui/assets/90650827/8d13f613-f529-4a3a-81a1-01c5cec3e39e)